playbooks/adhoc/docker_storage_to_overlay2/docker_storage_to_overlay2.yml

```diff
@@ -0,0 +1,103 @@
+---
+# This playbook changes the Docker storage driver to overlay2.
+#
+# Note, this makes no attempt to migrate existing containers/images.
+# It momentarily stops docker.service and uses "atomic storage reset"
+# to wipe all local Docker storage.
+#
+# To run:
+#
+# ansible-playbook -e 'cli_tag_name=<tag-name>' docker_storage_to_overlay2.yml
+#
+
+- name: Transition Docker storage driver to overlay2
+  hosts: "tag_Name_{{ cli_tag_name }}"
+  user: root
+  connection: ssh
+
+  # For 'ansible_distribution*'
+  gather_facts: yes
+
+  vars:
+    service_names:
+    - "docker.service"
+    optional_service_names:
+    - "atomic-openshift-node.service"
+
+  tasks:
+
+    # XXX Safe to assume atomic is installed?
+
+    - name: Start docker.service for info
+      service:
+        name: docker
+        state: started
+
+    - name: Check if storage driver is already overlay2
+      shell: docker info | grep --ignore-case 'storage driver:.*overlay2'
```

Comment:
We try really hard to not use `shell`. It's a very, very powerful module, which security wise is a bit scary.

More information from the openshift-ansible best practices guide:
https://github.com/openshift/openshift-ansible/blob/master/docs/best_practices_guide.adoc#The-Ansible-command-module-SHOULD-be-used-instead-of-the-Ansible-shell-module

Could we instead use `command`?

Something like:
```yaml
  - name: Check if storage driver is already overlay2
    command: docker info
    register: storage_driver_check
    changed_when: False
    ignore_errors: yes

  - name: Transition storage driver to overlay2
    when: storage_driver_check.stdout | lower | search('storage driver:.*overlay2')
    block:
[... snip ...]
```

Reply:
Ah ha, thanks for the link.  I'll rework to avoid `shell`.